integration/common.go

```diff
@@ -0,0 +1,101 @@
+// +build linux
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package integration
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/pelletier/go-toml"
+	"github.com/sirupsen/logrus"
+	cri "k8s.io/cri-api/pkg/apis"
+)
+
+// ImageList holds public image references
+type ImageList struct {
+	Alpine          string
+	BusyBox         string
+	Pause           string
+	VolumeCopyUp    string
+	VolumeOwnership string
+}
+
+var (
+	imageService cri.ImageManagerService
+	imageMap     map[int]string
+	imageList    ImageList
+	pauseImage   string // This is the same with default sandbox image
+)
+
+func initImages(imageListFile string) {
+	imageList = ImageList{
+		Alpine:          "docker.io/library/alpine:latest",
+		BusyBox:         "docker.io/library/busybox:latest",
+		Pause:           "k8s.gcr.io/pause:3.5",
+		VolumeCopyUp:    "gcr.io/k8s-cri-containerd/volume-copy-up:1.0",
```

Comment:
I think copy-up has a 2.0 just like the following volume-ownership?

Reply:
Right, I didn't notice on the file conflict.